addons/event/views/event_views.xml

```diff
@@ -44,7 +44,7 @@
                                             <field name="event_type_mail_ids" style="width: 100%;">
                                                 <tree string="Communication" editable="bottom">
                                                     <field name="notification_type" invisible="1"/>
-                                                    <field name="template_id" attrs="{'required': [('notification_type', '=', 'mail')]}"/>
```

Comment:
not required anymore ? (same for next usages)

Reply:
the field is always required now, so I added `required=True` on the field instead :)